src/PrestaShopBundle/Controller/Admin/AuthorizationController.php

```diff
@@ -0,0 +1,325 @@
+
+        if (!$adminDirectoryRenamed) {
+            try {
+                $newAdminDirectoryName = $adminDirectoryRenamer->renameToRandomName();
+
+                return $this->redirect(sprintf(
+                    '%s/%s/%s/',
+                    $tools->getShopDomainSsl(true),
+                    $newAdminDirectoryName,
+                    $this->generateUrl('_admin_login', [], UrlGeneratorInterface::RELATIVE_PATH)
+                ));
+            } catch (UnableToRenameAdminDirectoryException $e) {
+                $newAdminDirectoryName = $e->getDestinationName();
+            }
+        }
+
+        if ($secureModeChecker->isSslActivated() && !$secureModeChecker->isSslUsed()) {
+            $isInsecureMode = true;
+            $canAccessInsecureMode = $secureModeChecker->canIpAccessInsecureMode($request->getClientIp());
+        }
+
+        $templateVars += [
+            'shopName' => $this->configuration->get('PS_SHOP_NAME'),
+            'prestashopVersion' => $this->configuration->get('_PS_VERSION_'),
+            'imgDir' => $this->configuration->get('_PS_IMG_'),
+            'languageIso' => $languageDataProvider->getLanguageIsoById(
+                $this->configuration->get('PS_LANG_DEFAULT')
+            ),
+            'currentYear' => (new DateTime())->format('Y'),
+            'showLoginForm' => true,
+            'showForgotPasswordForm' => false,
+            'showResetPasswordForm' => false,
+            'isInsecureMode' => $isInsecureMode,
+            'canAccessInsecureMode' => $canAccessInsecureMode,
+            'secureUrl' => $secureModeChecker->secureUrl(
+                $this->generateUrl('_admin_login', [], UrlGeneratorInterface::ABSOLUTE_URL)
+            ),
+            'installDirectoryExists' => $installDirectoryExists,
+            'adminDirectoryRenamed' => $adminDirectoryRenamed,
+            'newAdminDirectoryName' => $newAdminDirectoryName,
+            'newAdminDirectoryUrl' => sprintf(
+                '%s%s%s',
+                $tools->getShopDomainSsl(true),
+                $this->configuration->get('__PS_BASE_URI__'),
+                $newAdminDirectoryName
+            ),
+        ];
+
+        if (!$adminDirectoryRenamed || $installDirectoryExists) {
+            $templateVars['showLoginForm'] = false;
+            $templateVars['showForgotPasswordForm'] = false;
+            $templateVars['showResetPasswordForm'] = false;
+        }
+
+        return $this->render('@PrestaShop/Admin/Login/index.html.twig', $templateVars);
+    }
+
+    /**
+     * Get all authorization error messages.
+     *
+     * @return array
+     */
+    private function getErrorMessages()
+    {
+        $employeeDoesNotExistMessage = $this->trans(
+            'The employee does not exist, or the password provided is incorrect.',
+            'Admin.Login.Notification'
+        );
+
+        $missingInformationMessage = $this->trans(
+            'Some identification information is missing.',
```

Comment:
It is a bit of an alarming tone, how about _Information needed_ instead? Do we help the user completing his information after that?

Reply:
@LouiseBonnard we don't really help the user to complete missing information, because it can be missing only if the page was malformed, not by something that depends on user input. 
So it basically means that the user won't be able to complete his request as something is wrong with the page.

I think we need a better wording for this, maybe you could help me with that? :slightly_smiling_face:

Reply:
How about _It looks like something went wrong, please try again._, unless we have an element of solution to give?

Reply:
the solution would be to start the password reset process from the beginning :slightly_smiling_face: